site-scrapers/MercyMedicalCenter/index.js

```diff
@@ -0,0 +1,95 @@
+const { site } = require("./config");
+const moment = require("moment");
+const s3 = require("../../lib/s3");
+const { sendSlackMsg } = require("../../lib/slack");
+const html_parser = require("node-html-parser");
+const { JSDOM } = require("jsdom");
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${site.name} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${site.name} done.`);
+    return {
+        ...site,
+        ...webData,
+        timestamp: new Date(),
+    };
+};
+
+async function jQueryPost(page, url, data) {
+    return await page.evaluate(
+        async (url, data) => {
+            return await new Promise((resolve) => {
+                $.post(url, data, function (data) {
```

Comment:
You may want to use fetch here since it is guaranteed to be on the page. Example in Walgreens.

Reply:
Per convo, jQuery post send data as FormData, which they are expecting in this case. `fetch` works nicely with json post bodies.